recipe/meta.yaml

```diff
@@ -1,12 +1,13 @@
-{% set version = "3.0.9" %}
+{% set name = "openpyxl" %}
+{% set version = "2.2.6" %}
```

Comment:
This is not an update. We can try a one-time PR to a branch to create an older release but without some context on badly this is needed it is probably not worth the effort.